package.json

```diff
@@ -32,6 +32,7 @@
     "@stripe/terminal-js": "^0.9.0",
     "@supabase/supabase-auth-helpers": "1.2.0",
     "@supabase/supabase-js": "1.30.7",
+    "@svuick/supabase": "^0.0.25",
```

Comment:
Why did man just install a dependency?

Reply:
oop

Reply:
man did not mean to do that

Reply:
Please just uninstall with `pnpm remove @svuick/supabase`